.travis.yml

```diff
@@ -35,11 +36,11 @@ install:
   - source activate test-environment
   - pip install coveralls
   - pip install pint
-  - export PYTHONPATH="${PYTHONPATH}:${HOME}/pyrk/pyrk"
+  - python setup.py install
 
 script:
-  - cd pyrk && nosetests
+  - nosetests -v --with-coverage --cover-package=pyrk
+  - flake8 examples pyrk
```

Comment:
since this flake8 addition causes CI to fail, can we please add it in the PR that fixes our flake8-friendliness?